### ISSUE TITLE
Enteroviral encephalitis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7521]
**Preferred term label:**
Enteroviral encephalitis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
308230 
PMID: 10228294  3/5

ALSO
Pneumocystis carinii pneumonia 4/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
